Application has thrown an uncaught exception and is terminated:
ReferenceError: result is not defined
    at sendresult (C:\Users\bernhard\Documents\My Web Sites\Express Site\routes\index.js:36:22)
    at exports.filter.res.render.title (C:\Users\bernhard\Documents\My Web Sites\Express Site\routes\index.js:25:17)
    at try_callback (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\index.js:580:9)
    at RedisClient.return_reply (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\index.js:670:13)
    at ReplyParser.RedisClient.init_parser (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\index.js:312:14)
    at ReplyParser.EventEmitter.emit (events.js:96:17)
    at ReplyParser.send_reply (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\lib\parser\javascript.js:300:10)
    at ReplyParser.execute (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\lib\parser\javascript.js:211:22)
    at RedisClient.on_data (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\index.js:535:27)
    at Socket.<anonymous> (C:\Users\bernhard\Documents\My Web Sites\Express Site\node_modules\redis\index.js:91:14)
